### DISCUSSION TITLE
新手向 Move 语言笔记

### DISCUSSION THREAD

**Discussion by Demian101** [NonceGeek/Move-Camp#147] [Materials]
撰写人背景：Web3 萌新，无 Rust 经验，无 Solidity 经验，没搞过 NFT，没做过 Dapp

笔记是跟着 叶开老师的 Bilibili [AptosEden](https://space.bilibili.com/88461412) 学习整理的 

ps: 如有错误恳请指正，感谢 Thanks♪(･ω･)ﾉ

**Comment by Demian101** [comment]
[Move 基础语法.pdf](https://github.com/NonceGeek/Web3-dApp-Camp/files/9845825/Move.pdf)

**Reply by Demian101** [reply]
Here

**Reply by leeduckgo** [reply]
Cool!

**Reply by SidWoong** [reply]
优秀！

**Reply by SidWoong** [reply]
https://explorer.aptoslabs.com/txn/12341396

请继续支持Aptos Eden

加油！！！

**Reply by JianiWang** [reply]
太有帮助了

**Comment by leeduckgo** [comment]
paid 10 xUSDT for this contri:

> https://stcscan.io/main/transactions/detail/0x2a2079be6e3191cdce069b3d81bab9bce2fa014b50b0363863920d2bca27b950
